modules/components/src/utils/parseInputFiles.js

```diff
@@ -0,0 +1,13 @@
+export default ({ files }) =>
+  Promise.all(
+    [...files].map(
```

Comment:
if `[...files]` is meant to catch nulls, maybe `(files || [])`... `[...null]` would throw an error anyway

Reply:
this is an interesting one, `files` isn't actually an array, it's an enumerable: https://developer.mozilla.org/en-US/docs/Web/API/FileList

enumerables are spreadable, so spreading into an array turns it into an array, and then you can map :)

Reply:
oh I see! yeah totally makes sense then :)